### PR TITLE
:chart_with_upwards_trend: add type for reposition table logging

### PR DESCRIPTION
**Pull request by hoshinotsuyoshi**
## Summary

add type for reposition table logging

(updated)

https://github.com/user-attachments/assets/8b279051-d36b-469b-b926-ca9341837fe7

### CODE REVIEW

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx [hunk 1]
```diff
@@ -113,6 +115,15 @@ export const ERDContentInner: FC<Props> = ({
     setNodes(updatedNodes)
   }, [edges, nodes, setNodes, setEdges, activeTableName])
 
+  const handleDragStopNode: OnNodeDrag<Node> = useCallback(
+    (_event, _node, nodes) => {
+      if (nodes.length === 1 && nodes[0]) {
```
[the remark this answers is not in the document]
Comment (junkisai) [reply frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx:120]: It seems necessary to design the log definition side also considering the ability to move multiple TableNodes together. 💭

File: frontend/packages/erd-core/src/features/gtm/utils/repositionTableLogEvent.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { pushToDataLayer } from './pushToDataLayer'
+
+type repositionTable = {
```
Comment (junkisai) [inline frontend/packages/erd-core/src/features/gtm/utils/repositionTableLogEvent.ts:3]: [nits]
It seems better to capitalize the first letter of type definitions.

```suggestion
type RepositionTable = {
```

**Review by junkisai — COMMENTED** [review]
It looked good, but I just commented on a very minor point. 🙏

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (hoshinotsuyoshi) [reply frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx]: > move multiple TableNodes together. 💭

Implemented! 

updated test video.
https://github.com/liam-hq/liam/pull/330#issue-2750051181

**Review by junkisai — APPROVED** [review]
That looks good 🎉 
Thank you for accommodating the log specification changes as well!

**Pull request merged**